Code/Conviva/Source/Events/Observers/AppEventForwarder.swift

```diff
@@ -11,7 +11,7 @@ import AVFoundation
 
 fileprivate let willEnterForeground = UIApplication.willEnterForegroundNotification
 fileprivate let didEnterBackground = UIApplication.didEnterBackgroundNotification
-fileprivate let newAccessLogEntry = NSNotification.Name.AVPlayerItemNewAccessLogEntry
+fileprivate let newAccessLogEntry = AVPlayerItem.newAccessLogEntryNotification
```

Comment:
what behavior change did this cause that makes reporting correct? (e.g. did you get the other event too late? is the underlying notification name different from the other? old event is not firing due to it was removed?)

Reply:
The other name was deprecated from iOS 17.2, so access log notifications were broken, so bitrate reporting was broken

Reply:
https://developer.apple.com/documentation/foundation/nsnotification/name/4108588-avplayeritemnewaccesslogentry